internal/types/alertlogs.go

```diff
@@ -163,3 +168,138 @@ func (f *AlertLogFilter) GetOffset() int {
 	}
 	return f.QueryFilter.GetOffset()
 }
+
+type AlertSettings struct {
+	Critical     *AlertThreshold `json:"critical"`
+	Warning      *AlertThreshold `json:"warning"`
+	AlertEnabled *bool           `json:"alert_enabled"`
+}
```

Comment:
sugest to not keep these as pointers else it becomes very difficult to read values in DB and it anyways is not adding special value

need to keep inputs in DTOs as pointers